tracer/test/Datadog.Trace.Tests/PublicApiTests.cs

```diff
@@ -11,5 +16,14 @@ public PublicApiTests()
             : base(typeof(Tracer).Assembly)
         {
         }
+
+#if NETFRAMEWORK
+        [Fact]
+        public void ExposesTracingHttpModule()
+        {
+            var httpModuleType = Type.GetType("Datadog.Trace.AspNet.TracingHttpModule, Datadog.Trace.AspNet");
```

Comment:
Good one. I forgot this was a thing. And then I spent 10 minutes looking for it on nuget (it's only published in the msi).

Reply:
This is one of the things that only comes to mind once in a blue moon, so I gotta write a test while it's top of mind! 😄